filebeat/crawler/registrar.go

```diff
@@ -212,28 +199,15 @@ func (r *Registrar) getPreviousFile(newFilePath string, newFileInfo os.FileInfo)
 }
 
 func (r *Registrar) setState(path string, state *FileState) {
```

Comment:
These getters and setters do not seem to be needed any longer. I'd just just remove them.